tests/Makefile

```diff
@@ -102,4 +102,4 @@ test-subset: create-venv
 
 # Run a subset of tests (comma-separated list) for CI
 ci-tests: TESTS=$(TESTS_CI)
-ci-tests: test-subset
```

Comment:
I think this is just a new empty line. Correct?

Reply:
Yes, the test-subset is in the latest line in the file